how-to/register-with-browser/client/src/launchbar.ts

```diff
@@ -0,0 +1,121 @@
+import { fin } from "openfin-adapter/src/mock";
+import { BrowserCreateWindowRequest, BrowserButtonType, BrowserWindowModule, getCurrentSync, Page, ToolbarOptions, WorkspacePlatformModule, PageLayout } from '@openfin/workspace-platform';
+import { createPageWithLayout, createViewIdentity } from "./browser";
+
+const platform: WorkspacePlatformModule = getCurrentSync();
+const defaultPageLayout: PageLayout = {
+    content: [
+        {
+            type: 'stack',
+            content: [
+                {
+                    type: 'component',
+                    componentName: 'view',
+                    componentState: {
+                        identity: createViewIdentity(fin.me.uuid, 'v1'),
+                        url: 'https://examples.com'
+                    }
+                },
+                {
+                    type: 'component',
+                    componentName: 'view',
+                    componentState: {
+                        identity: createViewIdentity(fin.me.uuid, 'v2'),
+                        url: 'http://openfin.jira.com'

```

Comment:
Can we use openfin.co instead of our jira url?

Reply:
changed to openfin.co